features/lima/README.md

```diff
@@ -4,17 +4,30 @@
 This feature flag produces an image suitable for using with [lima](https://lima-vm.io)
 </website-feature>
 
-Garden Linux images for Lima are published at https://images.gardenlinux.io
+Garden Linux images for Lima are published with nightly or release
 
 How to use the pre-built image:
 
-1. Create and start the VM
+1. Prerequisities
+    - We use glrd to get image link, this needs 
+        - sudo access
```

Comment:
This is only needed on mac I guess, on Linux this runs on user permissions. I would remove it or mention the mac part.

@SujanaSubr You could even try removing sudo from `bin/glrd` ... I have no idea how permissions regarding podman work on mac.